### DISCUSSION TITLE
Geochemical plots

### DISCUSSION THREAD

**Discussion by AbdulGhaffar396** [GeoPyTool/GeoPyTool#21] [General]
Hi, I need guidance for Geoplots, I am doing Ph.D. research in Petrology, could we work together for plots, it will be great for me.i have data and preparing for the publications. Thanks.

**Comment by ThirstyGeo** [comment]
@AbdulGhaffar396 you may be interested to use the pyrolite package instead?

[https://pyrolite.readthedocs.io/en/main/examples/index.html](https://pyrolite.readthedocs.io/en/main/examples/index.html)

**Reply by cycleuser** [reply]
Cool, nice package!

**Comment by cycleuser** [comment]
The usage of GeoPyTool can be viewed at http://geopytool.com/demonstration.html 

Besides, I am kind of tired of dealing with ancient plots such like about 30 years ago.
These old routines might also be included as @ThirstyGeo 's packages.

If you need more new plot functions, I am happy to write them for you when I got time.
The reference of the plot you need and some sample data are needed.
